provider/src/main/java/com/cloudera/director/google/internal/GoogleCredentials.java

```diff
@@ -73,6 +77,35 @@ private Compute buildCompute() {
     }
   }
 
+  private SQLAdmin buildSQLAdmin() {
+    try {
+      JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
+      HttpTransport httpTransport = GoogleNetHttpTransport.newTrustedTransport();
+      GoogleCredential credential;
+
+      if (jsonKey != null) {
+        credential = GoogleCredential.fromStream(
+            new ByteArrayInputStream(jsonKey.getBytes()), httpTransport, JSON_FACTORY)
+            .createScoped(Collections.singleton(SQLAdminScopes.SQLSERVICE_ADMIN));
+      } else {
+        Collection SQLSERVICE_ADMIN_SCOPES = Collections.singletonList(SQLAdminScopes.SQLSERVICE_ADMIN);
+
+        credential = GoogleCredential.getApplicationDefault(httpTransport, JSON_FACTORY);
+        credential = credential.createScoped(SQLSERVICE_ADMIN_SCOPES);
```

Comment:
Combine with previous line? Seems a bit strange to initialize the same variable on two consecutive lines.